### ISSUE TITLE
Missing application details

### ISSUE THREAD

**Issue by Uax1** [tora-tool/tora#127]
The exe file does not contain the details preventing it to be managed by 3rd party tool

![image](https://user-images.githubusercontent.com/48282743/53879751-21f27f80-4007-11e9-8a9f-59ddbfc43442.png)

**Comment by ibre5041** [comment]
Yeah. I used to sign .msi (.exe) by code signing certificate from https://www.certum.eu/. But the certificate expired, they doubled the price and request additional money for chip card and card reader. But still they seem to be cheepest provider of code signing certificates. So now I have doubts, whether Tora have so enought users, and whether is it worth of investing 200E into it.
